packages/wdio-utils/src/envDetector.ts

```diff
@@ -88,7 +88,8 @@ function isFirefox(capabilities?: Capabilities.DesiredCapabilities) {
  */
 function isMobile(capabilities: WebdriverIO.Capabilities) {
     const browserName = (capabilities.browserName || '').toLowerCase()
-
+    const bsOptions = capabilities['bstack:options'] || {}
+    const browserNameBS = (bsOptions.browserName || '').toLowerCase()
```

Comment:
I'd use the full name as `bs` isn't very descriptive for new contributors that aren't too familiar with it
```suggestion
    const browserstackBrowserName = (bsOptions.browserName || '').toLowerCase()
```